python/test/unit/fem/test_dofmap.py

```diff
@@ -469,60 +469,122 @@ def test_readonly_view_local_to_global_unwoned(mesh):
 
 
 @skip_in_parallel
-def test_high_order_lagrange():
-    """Test simple P3 Lagrange dofmap. Checks that dofs on a shared edged match."""
-    def check(mesh, edges):
-        """Compute the physical coordinates of the dofs on the given local edges"""
-        V = FunctionSpace(mesh, ("Lagrange", 3))
-
-        assert len(edges) == 2
-        dofmap = V.dofmap
-        dofs = [dofmap.cell_dofs(c) for c in range(len(edges))]
-        edge_dofs_local = [dofmap.dof_layout.entity_dofs(1, e) for e in edges]
-        for edofs in edge_dofs_local:
-            assert len(edofs) == 2
-        edge_dofs = [dofs[0][edge_dofs_local[0]], dofs[1][edge_dofs_local[1]]]
-        assert set(edge_dofs[0]) == set(edge_dofs[1])
+@skip_in_parallel
+@pytest.mark.parametrize('space_type', [
+    ("P", 1), ("P", 2), ("P", 3), ("P", 4),
+    ("N1curl", 1),
+    pytest.param(("N1curl", 2), marks=pytest.mark.xfail),
```

Comment:
Why are these expected to fail?

Reply:
There's a bug in ffc that is causing these to fail

Reply:
This bug (just added): https://github.com/FEniCS/ffcx/issues/175